python/ray/function_manager.py

```diff
@@ -437,6 +446,20 @@ def get_execution_info(self, driver_id, function_descriptor):
             A FunctionExecutionInfo object.
         """
         function_id = function_descriptor.function_id
+        if self._worker.load_code_from_local:
+            # For local code driver id is not necessary.
+            driver_id = ray.DriverID.nil()
+            try:
+                if len(function_descriptor.class_name) == 0:
+                    # This is a normal remote function.
+                    self._load_function_from_local(function_descriptor)
+                    self._num_task_executions[driver_id][function_id] = 0
+                # For actor, it should be loaded before this function.
+                return self._function_execution_info[driver_id][function_id]
+            except AttributeError as e:
+                logger.warning("Cannot find function {} in local files,"
+                               " with message: {} turn to GCS".format(
+                                   function_descriptor.function_name, e))
```

Comment:
No need to fallback to GCS lookup. Because when `load_code_from_local == True`, we don't export functions to GCS.
By doing so, the `if` and `else` branches can also be more consolidated.